hack/install/install_criu.sh

```diff
@@ -48,6 +48,7 @@ main() {
   os_dist="$(detect_os)"
   if [[ "${os_dist}" = "Ubuntu" ]]; then
```

Comment:
could we add one comment here?

Reply:
I do not think so, `>>>> install criu <<<<` will tell begin to download dependencies. and `>>>> start to download criu from github repository <<<<` will tell begin to download from github. Then we can know which step stack.